pep-0684.rst

```diff
@@ -516,24 +524,33 @@ there is no change in behavior under multiple interpreters created
 using the existing ``Py_NewInterpreter()``.
 
 Keep in mind that some extensions already break when used in multiple
-interpreters, due to keeping module state in global variables.  They
+interpreters, due to keeping module state in global variables (or
+due to the `internal state of linked libraries`_).  They
 may crash or, worse, experience inconsistent behavior.  That was part
 of the motivation for :pep:`630` and friends, so this is not a new
 situation nor a consequence of this proposal.
 
+.. _internal state of linked libraries: https://github.com/pyca/cryptography/issues/2299
+
 In contrast, when the `proposed API <proposed capi_>`_ is used to
 create multiple interpreters, the default behavior will change for
 some extensions.  In that case, importing an extension will fail
 (outside the main interpreter) if it doesn't indicate support for
 multiple interpreters.  For extensions that already break in
 multiple interpreters, this will be an improvement.
 
+Additionally, some extension modules link against libraries with
+thread-unsafe internal global state.
+(See `Restricting Extension Modules`_ above.)
+Such modules will have to start wrapping any direct or indirect use
+of that state in a lock.
+
 Now we get to the break in compatibility mentioned above.  Some
 extensions are safe under multiple interpreters, even though they
 haven't indicated that.  Unfortunately, there is no reliable way for
 the import system to infer that such an extension is safe, so
 importing them will still fail.  This case is addressed in
-`Extension Module Compatibility`_ below.
+`Extension Module Compatibility`_ above.
```

Comment:
BTW, being lazy and forgetful, I usually try to avoid having to update these references if I re-arrange sections by omitting a direction, and just stating e.g. "This case is address in :ref:`XXX`" or "the :ref:`XXX` section", since users can just click the link that points directly to the section in question. Also, I think I mentioned it before so I won't repeat that part, but another benefit of `:ref:` is it will automatically insert the section name and update in case you change its name, rather than having to hardcode it and manually update it everywhere.